pkg/apis/meta/v1/unstructured/converter.go

```diff
@@ -0,0 +1,204 @@
+/*
```

Comment:
Since this isn't dependent on metav1 should probably be in runtime.Unstructured, and referenced / used from here.  When I moved stuff here I did not explicitly separate out the generic bits from the specific bits, but no reason to add things here.